lib/ansible/plugins/strategy/__init__.py

```diff
@@ -1130,6 +1130,7 @@ def _evaluate_conditional(h):
 
         skipped = False
         msg = ''
+        self._tqm.send_callback('v2_playbook_on_task_start', task, is_conditional=False)
```

Comment:
This probably won't work, I'll have to copy it to each of the conditionals right below this line, and only send when we don't skip. As-is we send this even when given `when: false` on meta tasks that allow conditionals.